stories/atoms/components/canvas/Canvas.stories.mdx

```diff
@@ -60,10 +61,83 @@ export const CanvasStory = args => (
       height: 200
     }}
   >
-    {CanvasStory.bind({})}
+    {SimpleCanvas.bind({})}
   </Story>
 </Canvas>
 
 ### Properties
 
 <ArgsTable story="Overview" />
+
+## Auto resize
+
+Sometimes, you may want to resize the `Canvas` compopnent to fit the element or window that it’s in. To do this, simply bind the `resize` event and, once the event is caught,
```

Comment:
Oups, `compopnent` typo !

Reply:
Ah 🔬 good catch. Thx. Will be fixed.

Reply:
ok! fixed by [c77a85e](https://github.com/okp4/ui/pull/346/commits/c77a85eb49fbe276f93956bdf75d2df1c6165aaa).